tune_sklearn/tune_basesearch.py

```diff
@@ -405,6 +405,10 @@ def _fit(self, X, y=None, groups=None, **fit_params):
                 mode="max",
                 scope="last")
             self.best_params = self._clean_config_dict(best_config)
+            if not check_partial_fit(
```

Comment:
BTW, I think we should actually log something here. I think this behavior is somewhat of a "surprise", so being upfront that this methodology is being leveraged would be nice to mention.